modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -21,6 +21,10 @@
 	z-index: 1200;
 }
 
+.dropdown-menu.clay-color-dropdown-menu.show {
+	position: fixed;
+}
+
```

Comment:
Good job, @mlan0! :slightly_smiling_face: 

Can you please add another commit to this pull just to sort the CSS properties alphabetically?

At this moment, the properties `dropdown-*` aren't following the proper sorting.

Thanks in advance.

Reply:
Done!